metabase/metabase
# Make tooltip consistent for each icon

Closes https://github.com/metabase/metabase/issues/30945

### Description

In the notebook editor:
* change the placement of the tooltip of the `Show editor` tooltip to the top
* add tooltip to `Refresh` button

### How to verify

Go to any question or model, and see that `Show editor` tooltip is above the button, and the `Refresh` button now has a tooltip.

### Demo
![image](https://github.com/metabase/metabase/assets/25306947/c9be004d-e4f6-47a2-9e83-e352bbc004d2)
![image](https://github.com/metabase/metabase/assets/25306947/b7485d23-02d0-46fb-a6a5-882c1a9aa117)


### Checklist
